tools/universe_builder.py

```diff
@@ -120,103 +89,96 @@ def _get_file_template_mapping(self, filepath):
         template_mapping = {
             'package-version': self._pkg_version,
             'artifact-dir': self._upload_dir_url,
-            'jre-url': jre_url,
-            'jre-jce-unlimited-url': jre_jce_unlimited_url,
-            'libmesos-bundle-url': libmesos_bundle_url}
+            'jre-url': _jre_url,
+            'jre-jce-unlimited-url': _jre_jce_unlimited_url,
+            'libmesos-bundle-url': _libmesos_bundle_url}
 
         # look for any 'sha256:filename' template params, and get shas for those.
         # this avoids calculating shas unless they're requested by the template.
-        orig_content = open(filepath, 'r').read()
         for shafilename in re.findall('{{sha256:(.+?)}}', orig_content):
             # somefile.txt => sha256:somefile.txt
-            shafilepath = self._artifact_files.get(shafilename, '')
+            shafilepath = self._artifact_file_paths.get(shafilename, '')
             if not shafilepath:
                 raise Exception(
                     'Missing path for artifact file named \'{}\' (to calculate sha256). '.format(shafilename) +
-                    'Please provide the full path to this artifact (known artifacts: {})'.format(self._artifact_files))
+                    'Please provide the full path to this artifact (known artifacts: {})'.format(self._artifact_file_paths))
             template_mapping['sha256:{}'.format(shafilename)] = self._calculate_sha256(shafilepath)
 
         # import any custom TEMPLATE_SOME_PARAM environment variables:
         for env_key, env_val in os.environ.items():
             if env_key.startswith('TEMPLATE_'):
                 # 'TEMPLATE_SOME_KEY' => 'some-key'
-                template_mapping[env_key[9:].lower().replace('_','-')] = env_val
+                template_mapping[env_key[9:].lower().replace('_', '-')] = env_val
```

Comment:
9?  len('TEMPLATE_')?
Maybe not worth cleaning up here, but this line is doing way too much for my taste.